ser316asu-2018/Schwerin
# Us 24

I've realized that the wrong language/symbols are all hardcoded. So in this branch, I manually replace them with their supposed english translations.

TESTING INSTRUCTIONS
* Go through the diff and verify that all the replacement strings make sense
* Ensure you no longer see any signs of unicode symbols/non-english anywhere in the project
